com.ibm.streamsx.inet/impl/java/src/com/ibm/streamsx/inet/http/HTTPPostOper.java

```diff
@@ -182,7 +184,17 @@ public void setExtraHeaders(List<String> val) {
 	public void setAcceptAllCertificates(boolean val) {
 		this.acceptAllCertificates = val;
 	}
-	
+	@Parameter(optional=true, 
```

Comment:
Yes, the SPL compiler validates the attributes (not "attribute names"), I said this:

> The big advantage is that the SPL compiler will verify that the parameters exist ...

but should have said:

> The big advantage is that the SPL compiler will verify that the **attributes** exist ...